src/main/java/uk/gov/hmcts/cmc/claimstore/models/ServiceAddress.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.hmcts.cmc.claimstore.models;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.hibernate.validator.constraints.NotBlank;
+
+import java.util.Objects;
+import javax.validation.constraints.Size;
+
+public class ServiceAddress {
+    @NotBlank(message = "Service Address defendantsAddress should not be empty")
+    @Size(max = 3, message = "Service Address defendantsAddress should not be longer than {max} characters")
+    private final String defendantsAddress;
+
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    @Size(max = 100, message = "Address Line1 should not be longer than {max} characters")
+    private final String line1;
```

Comment:
Use an address class here, do not duplicate the code.
@JsonUnwrapped can be used to flatten the object so you don't get
```
serviceAddress:{
   address: {
  }
}
```